feature-group-definitions/field-sizing.yml

```
@@ -0,0 +1,2 @@
+name: field-sizing
+spec: https://drafts.csswg.org/css-ui-4/#field-sizing
```

This was new to me and I didn't know what it was. In the interest of making sense of this for others:

```suggestion
spec: https://drafts.csswg.org/css-ui-4/#field-sizing
description: The `field-sizing` CSS property sets whether an element may expand based on its contents. The property applies only to elements with a default preferred size, such as a `<textarea>`, which has a fixed size regardless of its content.
```